docs/sources/get-started/quick-start.md

```diff
@@ -2,160 +2,248 @@
-- **Gateway** (NGINX) which receives requests and redirects them to the appropriate container based on the request's URL.
+- **Gateway** (nginx) which receives requests and redirects them to the appropriate container based on the request's URL.
 - One Loki **read** component (Query Frontend, Querier).
 - One Loki **write** component (Distributor, Ingester).
 - One Loki **backend** component (Index Gateway, Compactor, Ruler, Bloom Compactor (Experimental), Bloom Gateway (Experimental)).
 - **Minio** an S3-compatible object store which Loki uses to store its index and chunks.
 - **Grafana** which provides visualization of the log lines captured within Loki.
 
-{{< figure max-width="75%" src="/media/docs/loki/get-started-flog-v3.png" caption="Getting started sample application" alt="Getting started sample application">}}
+{{< figure max-width="75%" src="/media/docs/loki/get-started-flog-v3.png" caption="Getting started sample application" alt="Getting started sample application" >}}
+
+<!-- Killercoda intro.md END -->
+
+## Before you begin
+
+Use the [Interactive Learning Environment](#interactive-learning-environment) to skip installing Docker and Docker Compose.
+
+- Install [Docker](https://docs.docker.com/install)
+- Install [Docker Compose](https://docs.docker.com/compose/install)
 
 ## Interactive Learning Environment
 
 {{< admonition type="note" >}}
-The Interactive Learning Environment is currently in trial. Please provide feedback, report bugs, and raise issues in the [Grafana Killercoda Repository](https://github.com/grafana/killercoda).
+The Interactive Learning Environment is in trial.
+
+Provide feedback, report bugs, and raise issues in the [Grafana Killercoda repository](https://github.com/grafana/killercoda).
 {{< /admonition >}}
 
-Try out this demo within our interactive learning environment: [Loki Quickstart Sandbox](https://killercoda.com/grafana-labs/course/loki/loki-quickstart) 
+Try out this demo within our interactive learning environment: [Loki Quickstart Sandbox](https://killercoda.com/grafana-labs/course/loki/loki-quickstart)
 
-- A free Killercoda account is required to verify you are not a bot.
-- Tutorial instructions are located on the left-hand side of the screen. Click to move on to the next section.
-- All commands run inside the interactive terminal. Grafana can also be accessed via the URL links provided within the sandbox.  
+- You must have a free Killercoda account to verify you aren't a bot.
+- Tutorial instructions are located on the left-side of the screen.
+  Click to move on to the next section.
+- All commands run inside the interactive terminal.
+- You can access Grafana with the URL links provided within the sandbox.
 
-## Installing Loki and collecting sample logs
+<!-- Killercoda step1.md START -->
 
-Prerequisites
+## Install Loki and collecting sample logs
 
-- [Docker](https://docs.docker.com/install)
-- [Docker Compose](https://docs.docker.com/compose/install)
+<!-- Killercoda ignore START -->
```

Comment:
Love this! I didn't think about specific sections we might want to ignore entirely